odt-opt-in/LETTERSPECS/BANNO.ODTOPTIN.V1.CFG

```diff
@@ -155,6 +162,47 @@ OTA:0034:300.00
 *  DEFAULT: NO (or left blank)
 COTA:
 *
+* ALLOW OVERDRAFT SETTINGS CHANGES IN MEMO MODE
+*  Line must begin with "MMOSC:"
+*  Should the program allow overdraft settings changes when the system is in memo mode?
+*  VALID VALUES: TRUE or FALSE
+*  DEFAULT:FALSE
+MMOSC:
+*
+* ERROR DISPLAY MESSAGES
+*  Line must be formatted as "EDMxxx:" where xxx represents the 3-digit error code that is returned to the UX.
+*  Up to 5 text lines can be defined per error code.
+*  VALID VALUES: "EDMxxx" where xxx is included in the list of supported error codes.
+*
+*  Supported Error Codes:
+*   501 - Config file read error
+*   502 - Config file validation error
+*   503 - No eligible shares to be created
+*   504 - Ineligible account type
+*   505 - Account warning exists
+*   506 - Error updating/creating share tracking
+*   507 - Error updating Auth/Fee fields
+*   508 - Error updating Overdraft Tolerance amount
```

Comment:
You have all of these as errors that support display messages but there are no calls to BNOERRDISPLAYLISTADD.  Here is an example of what that would look like if you are following the template framework:
```
 [* Set any errors that support display messages
 *]
 BNOERRDISPLAYCODE=BNOERRCODEINELIGIBLE
 CALL BNOERRDISPLAYLISTADD
 BNOERRDISPLAYCODE=BNOERRCODEINELIGIBLEACCT
 CALL BNOERRDISPLAYLISTADD
```

Reply:
All the requested changes made; does it pass code review now @rrobison44 ?